src/components/organizations/CreateOrganization.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+
+import ImageUploader from 'react-images-upload';
+
+export const fieldSetChildStyle = {
+    "display": "block",
+    "padding": "10px",
+    "margin": "10px auto 0",
+    "border-radius": "5px",
+    "border": "1px solid #a0a0a0"
+}
+
+export const inputStyle = {
+    "width": "400px",
+    "text-align": "center"
+}
+
+export const submitButtonStyle = { 
+    "color": "white",
+    "background-color": "rgba(255, 0, 0, 0.5)",
+    "font-weight": "bold",
+    "font-size": "20px",
+    "cursor": "pointer"
+}
+
+export const CreateOrganization = (props) => {
+
+    const updateNewOrgTitle = (event:React.ChangeEvent<HTMLInputElement>) => {
+        const newOrgTitle = event.target.value;
+        props.updateNewOrgTitle(newOrgTitle);
+    }
+
+    const updateNewOrgMission = (event:React.ChangeEvent<HTMLTextAreaElement>) => {
+        const newOrgMission = event.target.value;
+        props.updateNewOrgMission(newOrgMission);
+    }
+
+    const updateNewOrgUrl = (event:React.ChangeEvent<HTMLInputElement>) => {
+        const newOrgUrl = event.target.value;
+        props.updateNewOrgUrl(newOrgUrl);
+    }
+
+    const combineStyles = (...styles) => {
+        return Object.assign({}, ...styles);
+    }
+
+    //todo: force single image not working?
+
+    const updateNewOrgLogo = async (image) => {
+        const reader = new FileReader();
+        const file = image[0]
+        reader.onload = function(e:any) {
+            if(e.target) {
+                const src = e.target.result; 
+                file.src = src;
+                props.updateNewOrgLogo(file);
+            } 
+        };
+        reader.readAsDataURL(file);
+    }
+
+    //todo: ideally the state for this component is associated with the instance of it
+    console.log({addNewOrganization: props.addNewOrganization});
+
+    return <React.Fragment>
+        <ImageUploader
+            singleImage={true}
+            withPreview={true}
+            fileContainerStyle={combineStyles(fieldSetChildStyle, inputStyle)}
+            withLabel={false}
+            buttonText="Upload Organization Logo"
+            onChange={updateNewOrgLogo}
+            imgExtension={['.jpg', '.gif', '.png']}
+        />
+        <input style={combineStyles(fieldSetChildStyle, inputStyle)} id="orgName" type="text" placeholder="Organization Name" onChange={updateNewOrgTitle} value={props.newOrgTitle}/>
```

Comment:
Since u combine the same two styles thrice perhaps it is worth combining them earlier, and naming that

Reply:
fixed. now using spread operators within style={{}} to accomplish same thing. combineStyles() is dead.